example/src/App.js

```diff
@@ -1,22 +1,30 @@
-import React from 'react'
+import React from "react";
 
-import 'estella-two-column-dnd/dist/index.css'
+import "estella-two-column-dnd/dist/index.css";
 import { TwoColumnDnD } from "estella-two-column-dnd";
+import { Card, Typography } from "@material-ui/core";
 
 const getItem = index => {
   return {
     title: `title-${index}`,
     id: index
-  }
-}
+  };
+};
 
 const getItems = n => {
   return Array(n).fill(null)
-    .map((item, index) => getItem(index))
-}
+    .map((item, index) => getItem(index));
```

Comment:
Są potrzebne te średniki w jsie?

Reply:
Raczej nie, ale to tylko example, więc nie do końca zwracałem na to uwagę.